bundles/org.openhab.binding.velux/src/main/resources/ESH-INF/thing/channels.xml

```diff
@@ -0,0 +1,181 @@
+
+	<!-- Channel Type - of Binding -->
+
+	<channel-type id="information">
+		<item-type>String</item-type>
+		<label>@text/channel-type.velux.information.label</label>
+		<description>@text/channel-type.velux.information.description</description>
+		<category>Binding</category>
+		<state readOnly="true"></state>
+	</channel-type>
+
+
+	<!-- Channel Type - of Bridge -->
+
+	<channel-type id="status">
+		<item-type>String</item-type>
+		<label>@text/channel-type.velux.status.label</label>
+		<description>@text/channel-type.velux.status.description</description>
+		<category>Bridge</category>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="reload">
+		<item-type>Switch</item-type>
+		<label>@text/channel-type.velux.reload.label</label>
+		<description>@text/channel-type.velux.reload.description</description>
+		<category>Bridge</category>
+	</channel-type>
+
+	<channel-type id="timestamp" advanced="true">
+		<item-type>Number</item-type>
+		<label>@text/channel-type.velux.timestamp.label</label>
+		<description>@text/channel-type.velux.timestamp.description</description>
+		<category>Bridge</category>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="doDetection" advanced="true">
+		<item-type>Switch</item-type>
+		<label>@text/channel-type.velux.doDetection.label</label>
+		<description>@text/channel-type.velux.doDetection.description</description>
+		<category>Bridge</category>
+	</channel-type>
+
+	<channel-type id="firmware" advanced="true">
+		<item-type>String</item-type>
+		<label>@text/channel-type.velux.firmware.label</label>
+		<description>@text/channel-type.velux.firmware.description</description>
+		<category>Bridge</category>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="ipAddress" advanced="true">
+		<item-type>String</item-type>
+		<label>@text/channel-type.velux.ipAddress.label</label>
+		<description>@text/channel-type.velux.ipAddress.description</description>
+		<category>Bridge</category>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="subnetMask" advanced="true">
+		<item-type>String</item-type>
+		<label>@text/channel-type.velux.subnetMask.label</label>
+		<description>@text/channel-type.velux.subnetMask.description</description>
+		<category>Bridge</category>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="defaultGW" advanced="true">
+		<item-type>String</item-type>
+		<label>@text/channel-type.velux.defaultGW.label</label>
+		<description>@text/channel-type.velux.defaultGW.description</description>
+		<category>Bridge</category>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="DHCP" advanced="true">
+		<item-type>Switch</item-type>
+		<label>@text/channel-type.velux.DHCP.label</label>
+		<description>@text/channel-type.velux.DHCP.description</description>
+		<category>Bridge</category>
```

Comment:
This category does not exists, please check the [(old) list](https://docs.openhab.org/v2.2/concepts/categories.html)

The page on the new website is currently not there, [see the issue](https://github.com/openhab/openhab-docs/issues/1061)

Reply:
Thanks for the pointer. Migrated to categories:

> <category>Blinds</category>
> <category>NetworkAppliance</category>
> <category>WallSwitch</category>
> <category>Window</category>
>